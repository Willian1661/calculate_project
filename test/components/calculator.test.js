const calculator = require("../../src/components/calculator")

describe("calculator screen component", () =>{
    
    it("is passing calculator component", () => {
        expect(calculator("body-calc-id")).toMatchSnapshot()
    });

});
