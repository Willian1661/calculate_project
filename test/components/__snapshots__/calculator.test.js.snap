// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculator screen component is passing calculator component 1`] = `
"
    <div id="body-calc-id">

        <input type="text" placeholder="0" id="calculator-screen">

        <div id="calculator-buttons">
            
		<div id="buttons-top">
			<button>C</button>
			<button>+/-</button>
			<button>CE</button>
		</div>
            
		<div id="buttons-right">
			<button>%</button>
			<button>/</button>
			<button>*</button>
			<button>+</button>
			<button>-</button>
		</div>
            
		<div id="buttons-middle">
			<button>1</button>
			<button>2</button>
			<button>3</button>
			<button>4</button>
			<button>5</button>
			<button>6</button>
			<button>7</button>
			<button>8</button>
			<button>9</button>
		</div>
            
		<div id="buttons-bottom">
			<button>0</button>
			<button>.</button>
			<button>=</button>
		</div>
        </div>
    </div>
    "
`;
